__tests__/utilities/QuestioningAuthority.test.js

```diff
@@ -61,22 +58,78 @@ describe("createLookupPromise()", () => {
             label: "sweet corn",
           },
         ],
-      },
-      authLabel: "AGROVOC (QA)",
-      authURI: "urn:ld4p:qa:agrovoc",
-      label: "AGROVOC (QA)",
-      id: "urn:ld4p:qa:agrovoc",
-    }
+      }
+    },
+  }
+
+  beforeEach(() => {
+    global.fetch = jest.fn().mockImplementation(() => Promise.resolve(response))
+  })
+
+  describe("when authority with no subauthority", () => {
+    it("returns a promise from a search", async () => {
+      const authorityConfig = findAuthorityConfig("urn:ld4p:qa:agrovoc")
+      const result = await createLookupPromise("Corn", authorityConfig)
+      expect(global.fetch).toHaveBeenCalledTimes(1)
+      expect(global.fetch).toHaveBeenCalledWith(response.url)
+      expect(result.results).toHaveLength(8)
+    })
+  })
+
+  describe("when authority with subauthority", () => {
+    it("returns a promise from a search", async () => {
+      const authorityConfig = findAuthorityConfig("urn:ld4p:qa:geonames:water")
+      await createLookupPromise("Artic Sea", authorityConfig)
+      expect(global.fetch).toHaveBeenCalledWith(
+        "https://lookup.ld4l.org/authorities/search/linked_data/geonames_ld4l_cache/water?q=Artic+Sea&maxRecords=8&lang=en&context=true&response_header=true&startRecord=1"
+      )
+    })
+  })
+
+  describe("when non-LD authority", () => {
```

Comment:
should there be a test with subauthority for this one?   You mention 4 types in the class comments, so ... four cases in the test??

Reply:
This actually is an authority with subauthority. In addition, all of our non-LD authorities have subauthorities.

Made this clearer in test.